docs/cli.md

```diff
@@ -102,7 +102,7 @@ npx babel src --out-file script-compiled.js
 Ignore spec and test files
 
 ```sh
-npx babel src --out-dir lib --ignore spec.js,test.js
+npx babel src --out-dir lib --ignore src/**/*.spec.js,src/**/*.test.js
```

Comment:
Could you please add quotes around the double stars?

Otherwise, zsh will expand it before (I guess other Bashes too):
```sh
$ echo **/*.js
zsh: no matches found: **/*.js

# vs

$ echo "**/*.js"
**/*.js
```